### PR TITLE
OP-438 Update doc for new sms.settings

### PR DESCRIPTION
**Pull request by mwithi**
See OP-438.

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1523,76 +1522,76 @@ NOTE: An application restart is required to see the effect of any applied parame
 [#sms-properties]
 === 5.7 sms.properties
 
-Starting with Open Hospital version 1.8 a default sms.properties file is included:
+Starting with Open Hospital version 1.11 a default sms.properties file is included:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I believe we agreed not to note when some feature or function started as the documentation is meant for the current version of the product.   So I would remove `Starting with Open Hospital version 1.11 a default sms.properties file is included:` and replace it with `The default sms.properties file is:`.

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -1523,76 +1522,76 @@ NOTE: An application restart is required to see the effect of any applied parame
...
+* *TIMEOUT*: the timeout for the HTTP request. Required when sms.gateway is set to skebby-gateway-service or textbelt-gateway-service.
+* *LOOP*: the delay used by the sender to fetch new SMS operations. The value is in seconds.
+* *ICC*: the International Country Code that needs to be added to the phone numbers (if missing).
 
-[#gsm-properties]
-==== 5.7.1 GSM.properties
+==== 5.7.1 skebby-gateway-service
 
-Starting with Open Hospital version 1.8 a default GSM.properties file is included:
+Starting with Open Hospital 1.11 a default skebby-gateway-service section is included:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Similar comment as above.

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -1523,76 +1522,76 @@ NOTE: An application restart is required to see the effect of any applied parame
...
+* *password*: the password for authenticating to the Skebby server.
+* *ribbon.base-url*: the URL of the Skebby HTTP API.
+* *accessToken*: the token generated by the Skebby provider upon user request (it replaces username)
+* *userKey*: the key generated by the Skebby provider upon user request (it replaces password)
 
-==== 5.7.2 Skebby.properties
+==== 5.7.2 textbelt-gateway-service
 
-Starting with Open Hospital 1.8 a default Skebby.properties file is included:
+Starting with Open Hospital 1.11 a default textbelt-gateway-service section is included:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: And again the same comment

**Comment by dbmalkovsky** [comment]
Other than my comment it LGTM.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: I agree, but I searched the document and the notation was still there, so I just kept it for consistency.
I can change all occurrence at once.
Thanks!

**Review by goto-eof — APPROVED** [review]

**Pull request merged**